internal/config.go

```diff
@@ -64,15 +65,42 @@ func LoadConfig(path string) (*Config, error) {
 	return &c, nil
 }
 
-func (c *Config) fillAndValidate() error {
-	if c.Results.Common == nil {
-		if c.Results.FerretDB == nil || c.Results.MongoDB == nil {
-			return fmt.Errorf("both FerretDB and MongoDB results must be set (if common results are not set)")
+// mergeTestConfigs merges common config with both databases test configs.
```

Comment:
```suggestion
// mergeTestConfigs merges common config into both databases test configs.
```

To indicate direction